tests/haskell.vader

```diff
@@ -0,0 +1,39 @@
+Include: _setup.vader
```

Comment:
Oh.. I've meant to say `tests/ft/haskell.vader` - to keep maker related tests aside.